testcases/kdump/lib/crasher/crasher.c

```diff
@@ -99,6 +103,9 @@ int crasher_init(void)
 #endif
 	crasher_proc->read_proc = crasher_read;
 	crasher_proc->write_proc = crasher_write;
+#else
+	proc_create(CRASH, 0, NULL, &proc_fops);
+#endif
 	return 0;
 }
 
```

Comment:
Similar patch has been merged in 765857ba14, can you check if the module works for you now?